src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,13 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public String getPrintableString(Person p) {
+        String temp = "";
+        temp += p.getName().getPrintableString() + " ";
+        temp += p.getEmail().getPrintableString() + " ";
+        temp += p.getAddress().getPrintableString() + " ";
+        temp += p.getPhone().getPrintableString();
+        return temp; 
+    }
```

Comment:
The requirement was to prepare a "String getPrintableString(Printable... printables)" function though.